### ISSUE TITLE
Feature Request: Import/Export 1 bit and 2 bit sprite & tiles

### ISSUE THREAD

**Issue by Morokiane** [nesbox/TIC-80#1507]
Currently the only supported sprites and tiles that can be imported/exported are 4 bit. It would be a nice to have the ability to switch to 1-bit or 2-bit and then import/export the tiles/sprites. Another way would be in the terminal have a command such as

```
export tiles/sprites (1,2,4) file.png
import tiles/sprites (1,2,4) file.png
```

The 1,2,4 would be the desired bits to import/export the files as.

**Comment by ddelemeny** [comment]
I think this could be done as an external tool.

**Comment by PhilSwiss** [comment]
1. the *good* news:
    there is already an external tool for importing tiles & sprites, called **TicMcTile**, see https://github.com/PhilSwiss/ticmctile
2. the *bad* news:
    with the newest release (0.90.1706) of TIC80, the ability of loading scripts incl. data directly into the TIC has vanished. But an _investigation_ of this, is on the way. But you can still give it a try with [version 0.80.1344](https://github.com/nesbox/TIC-80/releases/tag/v0.80.1344) 

    By the way, Iam the author of **TicMcTile**, just for the record.

**Comment by joshgoebel** [comment]
>  the ability of loading scripts incl. data directly into the TIC has vanished. 

Wait what?

**Comment by PhilSwiss** [comment]
Ok, I'll clarify a few things here:
1. with the *old* TIC (v.80) you could start your script-code by typing: `tic80.exe myscript.lua`

2. with the *new* TIC (v.90) you can still run your script code with: `tic80.exe --fs=. --cmd="import code myscript.lua & run"`

So, loading and executing scripts works just fine, no need to worry.

But the main difference here is, that data which is encoded in comments, like:
```
-- <PALETTE>
-- 000:1a1c2c5d275db13e53ef...
-- </PALETTE>
```
Will be shown in the code-editor, but does NOT get used by the TIC as in the older versions. So importing *encoded* data like palettes, tiles, sprites, etc. is now longer possible.
But as said before, we have to dig into this a bit deeper and see if this can be fixed somehow. Or its getting replaced by another way to do it, we will see, hopefully.

**Comment by nesbox** [comment]
> So importing encoded data like palettes, tiles, sprites, etc. is now longer possible.

Carts in text format is a Pro feature since 0.90 (it was designed as a pro feature but was available for free on versions < 0.90)

**Comment by nesbox** [comment]
Import/export sprites could look like 
```
import sprites img.png bank=2 page=2 mode=4
```

**Comment by PhilSwiss** [comment]
> Carts in text format is a Pro feature since 0.90 (it was designed as a pro feature but was available for free on versions < 0.90)

Thanks for the clarification.



> ```
> import sprites img.png bank=2 page=2 mode=4
> ```
That looks promising, thumbs up.
